run.sh

```diff
@@ -4,6 +4,7 @@
 SCRIPTDIR="$( cd "$( dirname "${BASH_SOURCE[0]}" )" && pwd )"
 REPODIR="$(dirname "$SCRIPTDIR")"
 ARKSERVER=${ARKSERVER_SHARED:-"/ark/server"}
+VERBOSE=${VERBOSE:-"--verbose"}
```

Comment:
Sorry to nit-pick on this one. The default should be as before: no verbosity.

If this variable is only used for arkmanager, one could name it exactly like that. ARKMANAGER_VERBOSE true/false and later ARKMANAGER_ARGS="--verbose" if "ARKMANAGER_VERBOSE" = "true". Or even ARKMANAGER_ARGS, to be more generic for arguments to arkmanager (because that's what it's actually currently doing).

If more verbosity also affects other commands, it can be more generically named VERBOSE and used appropriately where needed.